include/mesh.h

```diff
@@ -0,0 +1,52 @@
+//
+// Created by Renjie Wu on 2019-03-07.
+//
+
+#ifndef PIPELINE_GRAPH_MESH_H
+#define PIPELINE_GRAPH_MESH_H
+
+#include "edge.h"
```

Comment:
These are after the standard includes: https://cb-geo.github.io/cpp-guide/#/header/order_includes